debug_toolbar/static/debug_toolbar/js/timer.js

```diff
@@ -28,36 +27,20 @@ function insertBrowserTiming() {
         } else {
             width = 0;
         }
-        return width < 1 ? "2px" : width + "%";
+        return width < 1 ? "2px" : `${width}%`;
     }
     function addRow(tbody, stat, endStat) {
         const row = document.createElement("tr");
         if (endStat) {
             // Render a start through end bar
-            row.innerHTML =
-                "<td>" +
-                stat.replace("Start", "") +
-                "</td>" +
-                '<td><svg class="djDebugLineChart" xmlns="http://www.w3.org/2000/svg" viewbox="0 0 100 5" preserveAspectRatio="none"><rect y="0" height="5" fill="#ccc" /></svg></td>' +
-                "<td>" +
-                (performance.timing[stat] - timingOffset) +
-                " (+" +
-                (performance.timing[endStat] - performance.timing[stat]) +
-                ")</td>";
+            row.innerHTML = `<td>${stat.replace("Start", "")}</td><td><svg class="djDebugLineChart" xmlns="http://www.w3.org/2000/svg" viewbox="0 0 100 5" preserveAspectRatio="none"><rect y="0" height="5" fill="#ccc" /></svg></td><td>${performance.timing[stat] - timingOffset} (+${performance.timing[endStat] - performance.timing[stat]})</td>`;
```

Comment:
I think this change and the one on line 43 are a step backwards in terms of maintainability.

Reply:
You're not objecting to the use of the template literal I hope, just to the fact that we're using too much inline code here and have lost the line wrapping? I'll fix that.

Reply:
I didn't even notice the template literal :sweat_smile: Yes, my issue is that the line of code is really long. I think the multiple lines helped convey what logic was occurring, then letting long static text exist where it needed to be.